### DISCUSSION TITLE
Problemas con MYSQL

### DISCUSSION THREAD

**Discussion by CarlosTochi** [Launch-X-Latam/MisionFrontEnd#598] [Team Posts]
Hola 
Alguien tuvo problemas de conexión con el servidor mysql flexible de azure??

![Selección_624](https://user-images.githubusercontent.com/78807139/211637133-4dc9abb5-2693-45dd-b626-5d3538a94bd9.png)

o Alguien sabe como solucionar el siguiente error??

**Comment by shelkanife** [comment]
Según la página de solución de errores ese error se produce si:
- El nombre de usuario no existe.
- El nombre de usuario se ha eliminado.
- Su contraseña se ha cambiado o restablecido.

Verifica tu nombre de usuario y contraseña

**Comment by CarlosTochi** [comment]
Sabes en donde puedo ver los usuarios que se defienieron??

**Comment by shelkanife** [comment]
En la infomación general de tu recurso se muestra el nombre del administrador del servidor y también la opción para restablecer la contraseña

**Comment by CarlosTochi** [comment]
Ya vi mi error, estaba utilizando caracteres no alfanuméricos como: ¿?#$ 

Con que utilicen caracteres mayúsculas, minúsculas y números es mas que suficiente

**Comment by lODIN007l** [comment]
> Sabes en donde puedo ver los usuarios que se defienieron??

podrias ver la shell de mysql dado que ya tienes la instancia
